schemas/samples/00001-jws-generic.md

```diff
@@ -0,0 +1,92 @@
+# JWS Generic Example
+
+Calculating JSON Web Signature (JWS) for a generic payload (MWE)
+A signature must be a valid Flattened JWS JSON serialised string. For instance a payload (serialised as string)
```

Comment:
I suggest we rename the term `payload` to `to-be-signed payload` for more clarity